lib/workers/repository/index.ts

```diff
@@ -17,10 +17,12 @@ import { ensureOnboardingPr } from './onboarding/pr';
 import { extractDependencies, updateRepo } from './process';
 import { ProcessResult, processResult } from './result';
 import { printRequestStats } from './stats';
+import type { SshSocket } from '../global/ssh_socket';
 
 // istanbul ignore next
 export async function renovateRepository(
   repoConfig: RenovateConfig,
+  sshSocket: SshSocket = undefined,
```

Comment:
```suggestion
  sshSocket: SshSocket | undefined = undefined,
```

Reply:
Done

Reply:
Please read https://github.com/renovatebot/renovate/blob/main/.github/contributing.md#resolve-review-comments-instead-of-commenting